modules/tinymce/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Search and replace plugin would incorrectly find matching text inside SVG elements. #TINY-10162
 - Removed use of `async` for editor rendering which caused visual blinking when reloading the editor in-place. #TINY-10249
 - List items containing a list element surrounded by non list nodes would cause some list operations to fail. #TINY-10268
+- Deleting a range that includes a part of a `table` and a `text`, both inside a `td` removed the entire content of the `td`. #TINY-10254
```

Comment:
Needs to be more user-focussed and past tense. Also it probably applies to `th` as well so just call it "cell"
```suggestion
- Deleting a range that included both text of a cell and a nested table in that cell removed the entire content of the cell. #TINY-10254
```

Reply:
changed